bundles/org.openhab.io.homekit/README.md

```diff
@@ -994,3 +994,32 @@ You can verify this with [Discovery DNS iOS app](https://apps.apple.com/us/app/d
   - if sf is equal 1, openHAB is accepting pairing from new iOS device. 
   - if sf is equal 0 (as on screenshot), openHAB is already paired and does not accept any new pairing request. you can reset pairing using `openhab:homekit clearPairings` command in karaf console.
 - if you see openHAB bridge and sf is equal 1 but you dont see openHAB in home app, probably you home app still think it is already paired with openHAB. remove your home from home app and restart iOS device.
+
+### Re-adding the OpenHAB HomeKit bridge reports that a bridge is already added, even though it has clearly been removed.
+
+There are various reasons this may happen. Try the following:
+
+* In [openhab-cli](https://www.openhab.org/docs/administration/console.html), run `openhab:homekit clearPairing`. Reboot openHAB. Reboot your iOS device. Try again.
```

Comment:
There should be no need (anymore) to reboot either device. https://github.com/openhab/openhab-addons/pull/14016 also fixes an issue where the bridge wasn't going back into discovery mode after unpairing.

Reply:
That fix should land in 3.4.1, right?